deno.json

```diff
@@ -10,7 +10,7 @@
     }
   },
   "tasks": {
-    "test": "deno test -A --unstable tests/ src/",
+    "test": "deno test -A --unstable --shuffle --parallel tests/ src/",
```

Comment:
Question: is this relevant to the issue?

Reply:
It's not, therefore I can split it in another branch and PR

Reply:
It is not, it was just an easy add-in. I have now pulled out the commit containing this addition.